### PR TITLE
Refactor to create config file if one does not exist already.

### PR DESCRIPTION
**Pull request by crashGoBoom**
Hi! This will create a dir and config file if they don't exist already.

Related to: https://github.com/99designs/aws-vault/issues/175

Please let me know if any changes should be made. Thanks!

### CODE REVIEW

File: vault/config.go [hunk 1]
```diff
@@ -38,6 +38,29 @@ func ConfigPath() (string, error) {
...
 
+// CreateConfig will create the config directory and file if they do not exist
+func CreateConfig() (error) {
+	file, err := ConfigPath()
+	if err != nil {
+		return err
+	}
+	dir := filepath.Dir(file)
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
+		os.Mkdir(dir, 0755)
```
Comment (lox) [inline vault/config.go]: 0700 probably makes sense for this?

**Comment by lox** [comment]
Thanks @crashGoBoom!

File: vault/config.go (hunk 1 above)
Comment (crashGoBoom) [reply vault/config.go]: @lox oops! Fixed in the latest push. Thanks!

**Comment by lox** [comment]
I'll merge as soon as it passes, thanks for fixing this @crashGoBoom, I've been meaning to for ages!

**Comment by crashGoBoom** [comment]
@lox No problem glad I could help!

**Comment by bhattraideb** [comment]
I have installed v4.2.0 as explained [here.](https://docs.cloudposse.com/tools/aws-vault/?__cf_chl_jschl_tk__=8366a6e166f4975a67c573756b7fb9382fd46bd8-1606230714-0-AWe6FwMl3v_uN98TgSlQ7n4RL0RxmCec--rrgEcTUhV6fCt9BJpyBykYunE3vY8TWArn1Y_LShbCdjfkcktjj8h85urVcm25Y1w5pfKCCSvzFPIlmEs0XM1EkpESHCXb9bUnElWxCtww1Y7QKaAqEIH0BHETnO-Ov2WoLGlIQlcjOt7ZJNTCKXwvd9Ryl1J3SfdlAohkdHTGMYN1uCx0O1YrBnViPnB4fUl8yBHUmsAcvAiYtM-manZUssuPiHDHFN30JzdCGjpVY7qizIKijsy3IEXLQ9Die0HpUSJdicWC3LNeFIndrk6sirLLukR2qgdXKM7dBm-0HeLr1ImSbH19jzeR4Ug-O2CHnGcqD65yYK9rtYkHiKZSXBTxOdR1onF2IaZPMq6i3KOVg2aE6GY) Is there any way to update to the latest version? I can see its latest version is [6.2.0](https://github.com/99designs/aws-vault/releases)

**Pull request merged**